.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -29,6 +29,36 @@ assignees: freitagb, wildintellect
 **Intended science use case**
 *Please describe how you intend to use the data, or the expected relevance to MAAP users.*
 
+## Spatio-Temporal Asset Catalog (STAC)
+
+This collection will be published as a Spatio-Temporal Asset Catalog (STAC) Collection. You can read the complete STAC collection spec here: (https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md).
+
+If you need help with any of the fields below, please let us know.
+
+**id:** Identifier for the Collection that is unique across the provider. This is typically an abbreviated and hyphenated or camel-cased version of the dataset name, For example `gedi_l2a_v002` for "GEDI L2A Elevation and Height Metrics Data Global Footprint Level V002".
+
+_We can create an id from the Dataset Name from the first section if not otherwise indicated._
+
+**title:** A short descriptive one-line title for the collection. Technically, this is not requried by the STAC Spec but it is used by the STAC browser.
+
+_We will use the Dataset Name from the first section if not otherwise indicated._
+
+**description:** Detailed multi-line description to fully explain the collection.
+
+_We will reuse the Dataset Description from the first section if not otherwise indicated._
+
+**spatial extent:** A bounding box for the potential spatial extents covered by the collection. Read more in the [spatial-extent-object section of the spec](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#spatial-extent-object).
```

Comment:
Since Spatial Extent and Temporal Extent are the most important can we move these to to the top of list?

Reply:
Also the "additional information" section at the bottom mentions spatial and temporal info again